### PR TITLE
[4.0.0] Update Okta doc to refer the new Okta UI

### PR DESCRIPTION
**Pull request by PasanT9**

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

File: en/docs/administer/key-managers/configure-okta-connector.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ Follow the instructions below to configure Okta as a third-party Key Manager:
 
 ## Step 1 - Configure Okta
 
-1. Navigate to the [Okta sign up page](https://developer.okta.com/signup/).
+1. Navigate to the [Okta sign up page](https://developer.okta.com/signup/) and singup to **Workforce Identity Cloud**. We have created an account in **Developer Edition** for this.
```
Comment (tharikaGitHub) [inline en/docs/administer/key-managers/configure-okta-connector.md:9]:
```suggestion
1. Navigate to the [Okta sign up page](https://developer.okta.com/signup/) and singup to **Workforce Identity Cloud**. 
```

File: en/docs/administer/key-managers/configure-okta-connector.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ Follow the instructions below to configure Okta as a third-party Key Manager:
...
 
      [![Okta signup]({{base_path}}/assets/img/administer/okta-signup.png)]({{base_path}}/assets/img/administer/okta-signup.png)
 
      Create an Okta account and get the Okta instance URL.
```
Comment (tharikaGitHub) [inline en/docs/administer/key-managers/configure-okta-connector.md:13]: Create an Okta account and get the Okta instance URL. You can create an account in **Developer Edition** for this.

Comment (tharikaGitHub) [reply en/docs/administer/key-managers/configure-okta-connector.md:13]: Shall we reword as above?

Comment (PasanT9) [reply en/docs/administer/key-managers/configure-okta-connector.md]: Fixed with https://github.com/wso2/docs-apim/pull/8155/commits/387ce46364500909c31063133f186538f9120a48

File: en/docs/administer/key-managers/configure-okta-connector.md (hunk 1 above)
Comment (PasanT9) [reply en/docs/administer/key-managers/configure-okta-connector.md]: Fixed with https://github.com/wso2/docs-apim/pull/8155/commits/387ce46364500909c31063133f186538f9120a48

**Review by HeshanSudarshana — APPROVED** [review]

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**